ephypype/tests/test_import_ctf.py

```diff
@@ -1,15 +1,23 @@
 """Test import_ctf."""
 import mne
-import os.path as op
+import tempfile
+import os
 
 from ephypype.import_ctf import convert_ds_to_raw_fif  # noqa
 
 data_path = mne.datasets.testing.data_path()
-ds_fname = op.join(data_path, 'CTF', 'testdata_ctf.ds')
+ds_fname = os.path.join(data_path, 'CTF', 'testdata_ctf.ds')
 
 
 def test_convert_ds_to_raw_fif():
     """Test data conversion."""
 
+    current_wd = os.getcwd()
+    tmp_dir = tempfile.mkdtemp()
+
+    os.chdir(tmp_dir)
```

Comment:
can you refactor this into a private function in `ephypype/utils.py` ?

Reply:
yes, I thought to do it :)